app/views/pages/moving-guide/claims.html.erb

```diff
@@ -0,0 +1,198 @@
+<%- @page_title = 'Lost & Damage Claims' -%>
+
+<%= render partial: 'shared/contextual_header', locals: {page_title: @page_title, parent_pages: []} %>
+
+<div class="usa-section main-section">
+  <h1><%= @page_title %></h1>
+
+  <aside class="sidenav">
+    <div class="sidenav-jump-to">
+      Jump to:
+    </div>
+    <ul class="usa-sidenav-list">
+      <li><%= link_to 'Lost & Damaged Household Goods Items', '#lost-damaged-items' %></li>
+      <li><%= link_to 'What does "Full Replacement" Value" mean?', '#full-replacement-value' %></li>
+      <li><%= link_to 'Loss & Damage Report', '#report' %></li>
+      <li><%= link_to 'Itemized Claim', '#claim' %></li>
+      <li><%= link_to 'Residential Damage', '#residential-damage' %></li>
+    </ul>
+  </aside>
```

Comment:
We're adding this sidenav pattern to a bunch of pages. We _could_ do something interesting with partials:

```erb
# app/views/shared/_sidenav.html.erb

<aside class="sidenav">
  <div class="sidenav-jump-to">
    Jump to:
  </div>
  <ul class="usa-sidenav-list">
    <%= yield %>
  </ul>
</aside>
```

```erb
# app/views/pages/moving-guide/claims.html.erb, etc. etc.

<%= render layout: 'shared/sidenav' do %>
  <li><%= link_to 'Lost & Damaged Household Goods Items', '#lost-damaged-items' %></li>
  <!-- etc. etc. etc. -->
<% end %>
```

Reply:
Oh! I was wondering how you do this with rails.